go.mod

```diff
@@ -22,7 +22,7 @@ require (
 	k8s.io/apimachinery v0.28.2
 	k8s.io/client-go v13.0.0+incompatible
 	k8s.io/helm v2.17.0+incompatible
-	k8s.io/klog/v2 v2.100.1 // indirect
+	k8s.io/klog/v2 v2.100.1
```

Comment:
Do you need to run `go mod tidy`? This doesn't look intentional.

Reply:
This is my go mod tidy result.. I can bring it back

Reply:
"k8s.io/klog/v2 should be directgo mod tidy" I am having this warning should I bring it back?

Reply:
No, it's fine--I was surprised since I didn't see it as an added import, but I guess it just hadn't been run previously.